### ISSUE TITLE
Async options in select are not reflected

### ISSUE THREAD

**Issue by shindekaps** [sherweb/ng2-materialize#88]
Ng for not working for select component

**Comment by jfcere** [comment]
Hi @shindekaps,

We had this issue previously but it is supposed to be fixed in v1.2.0. Here is a quick example ...

```html
<mz-select-container>
  <select mz-select
    id="select-country"
    [label]="'Country'"
    [placeholder]="'Choose your country'">
    <option *ngFor="let country of countries" [selected]="country === user.country" [ngValue]="country">{{ country.name }}</option>
  </select>
</mz-select-container>
```

If you have updated to v1.2.0 and still have the problem can you provide a sample of your code?

Thanks,

**Comment by shindekaps** [comment]
Hi,
*ngFor is working when data is hard coded. When the data is coming asynchronous then it not working. Also I got workaround when we use ngIf then it is working.
E.g.
  `   <mz-select-container>
  <select mz-select
    id="select-number"
    [label]="'Number'"
    [placeholder]="'Choose a number'"
    [(ngModel)]="optionValue">
    <option *ngFor="let option of comapnyList" [selected]="option === optionValue">{{ option }}</option>
  </select>
</mz-select-container>`

where companyList is coming from server.Above example is not working. But when we add *ngIf then it is working
E.g.
  `   <mz-select-container *ngIf="comapnyList.length>0">
  <select mz-select
    id="select-number"
    [label]="'Number'"
    [placeholder]="'Choose a number'"
    [(ngModel)]="optionValue">
    <option *ngFor="let option of comapnyList" [selected]="option === optionValue">{{ option }}</option>
  </select>
</mz-select-container>`

Thanks
Kapil Shinde

**Comment by charpeni** [comment]
Thank you for the reporting, you're right there're bugs with async options.

We're on it.

**Comment by charpeni** [comment]
@shindekaps it should be fixed by #91 and it will be released to npm in the next days/weeks.

If there is anything else or if the issue is not properly fixed, feel free to ping us.
